Switch: add getting started tutorial (#4446)
DevExpress/devextreme-documentation: concepts/05 UI Components/Switch/00 Getting Started with Switch/15 Configure Size and Representation.md

> The component allows you to configure its size and representation. To specify a width, assign the needed value to the [width](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#width) property. Use the [rtlEnabled](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#rtlEnabled) property to switch the widget to a right-to-left representation. If you set this property to `true`, the Switch label flows from right to left, and the layout of elements is reversed.

Reviewer: "representation" is a vague word - can mean pretty much anything. Moreover, we are advised to avoid the word "represent" where possible, because of its vagueness. 

In the end -  a lot of things here can be shortened. But I would also separate the two concepts into their own paragraphs, because they are pretty independent. 

```suggestion
The component allows you to change its [width](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#width). 

You can also use [rtlEnabled](/Documentation/ApiReference/UI_Components/dxSwitch/Configuration/#rtlEnabled) property to enable Right-to-Left layout. 
```

---
##### jQuery

    <!-- tab: index.js -->
    $(function() {
        $("#switch").dxSwitch({
            // ...
            width: 80,
            rtlEnabled: true,
        });
    });

##### Angular

    <!-- tab: app.component.html -->
    <dx-switch ...
        [width]="80"
        [rtlEnabled]="true"
    >
    </dx-switch>

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxSwitch ...
            :width="80"
            :rtlEnabled="true"
        />
    </template>

    <script>
    // ...
    </script>

##### React

    <!-- tab: App.js -->
    // ...
    
    function App() {
        // ...
        return (
            <Switch ...

>                 width="80"

Reviewer 2: Not a big deal but just for consistency better use `width={80}` here, because on others tabs width is set as number
